metricq/history_client.py

```diff
@@ -379,9 +389,12 @@ async def history_data_request(
                 See :class:`.HistoryRequestType`.
             timeout:
                 Operation timeout in seconds.
+
+        Raises:
+            ValueError: if
```

Comment:
👀

Reply:
Funny, this showed me that it was missing the check for 255 bytes :D